HlibPolishchuk/HW11/2.py

```diff
@@ -0,0 +1,25 @@
+def the_day_of_the_week():
+    try:
```

Comment:
another way:

def day_of_week(number):
    
    day_list = ["Monday", "Tue", "Wed", "Thu", "Fri", "Sat", "Sunday"]
    
    if type(number) is not int:
        raise ValueError("You input not number")
    
    if number<=0 or number>7:
        raise IndexError("Your number is out of range")
    
    print(day_list[number-1])
    

try: 
    day_of_week("gchgkc")   
    day_of_week(789)
    day_of_week(7)

except ValueError as e:
    print("Wrong Input!", e)
except TypeError as e:
    print("Wrong Input!", e)
except IndexError as e:
    print("Index Error", e)

day_of_week("gchfc")